[CI:DOCS] Add example of fuse-overlay to podman system reset (#8278)
containers/podman: docs/source/markdown/podman-system-reset.1.md

**podman system reset** [*options*]

## DESCRIPTION
**podman system reset** removes all pods, containers, images and volumes.

> This command Must be run **before** changing any of the following fields in the

Reviewer: ```suggestion
This command must be run **before** changing any of the following fields in the
```
or if you really want to emphasize it, bold it.

`containers.conf` or `storage.conf` files: `driver`, `static_dir`, `tmp_dir`
or `volume_path`.

> `podman system reset` reads the current configuration and attempt to remove all

Reviewer: ```suggestion
`podman system reset` reads the current configuration and attempts to remove all
```

> configuration. If the administrator modified the configuration files first,

Reviewer: ```suggestion
of the relevant configurations. If the administrator modified the configuration files first,
```

> podman system reset might not be able to cleanup the previous storage.

Reviewer: ```suggestion
`podman system reset` might not be able to clean up the previous storage.
```

**podman system reset** [*options*]

## DESCRIPTION
**podman system reset** removes all pods, containers, images and volumes. Must
be run after changing any of the following values in the `containers.conf` file:
`static_dir`, `tmp_dir` or `volume_path`.

> Note: If administrators wants to change the /etc/containers/storage.conf in any

Reviewer 2: Should we change this to "change any paths or the storage driver" instead of "change anything"? I'm pretty sure that setting ignore_chown_errors wouldn't need a `system reset` first, for example?

Author: I do you the term "may".  I think it is fairly difficult to get specific here.  I think they would only get here if something is majorly wrong.

Author: Ok I reworked a little bit more.

Print usage statement

## EXAMPLES

### Switching rootless user from VFS driver to overlay with fuse-overlayfs

If the user ran rootless containers without having the `fuse-overlayfs` program
installed, podman defaults to the `vfs` storage in their home directory. If they
want to switch to use fuse-overlay, they must install the fuse-overlayfs
package. The user needs to reset the storage to use overlayfs by default.
Execute `podman system reset` as the user first to remove the VFS storage. Now
the user can edit the `/etc/containers/storage.conf` to make any changes if

> necessary. If the systems default was already `overlay, then no changes are

Reviewer: ```suggestion
necessary. If the system's default was already `overlay`, then no changes are
```

> necessary to switch to fuse-overlayfs. Podman is looks for the existence of the

Reviewer: ```suggestion
necessary to switch to fuse-overlayfs. Podman looks for the existence of 
```

> fuse-overlayfs to use it when set in `overlay` driver, only falling back to vfs

Reviewer: ```suggestion
fuse-overlayfs to use it when set in the `overlay` driver, only falling back to vfs
```

> if the program does not exist. Users and run `podman info` to ensure Podman is

Reviewer: ```suggestion
if the program does not exist. Users can run `podman info` to ensure Podman is
```

> using fuse-overlayfs and overlay driver.

Reviewer: ```suggestion
using fuse-overlayfs and the overlay driver.
```
